source/_integrations/iotty.markdown

```diff
@@ -46,3 +57,7 @@ Each iotty device gets mapped into one Home Assistant device per gang, each with
 ### Switches
 
 The main, unnamed, switch entity controls the light switch, turning it on or off the gang related to it.
+
+### Covers
```

Comment:
the introducstion sentence in line 19, is it still accurate, now that we also have covers? should it be updated to include covers? Or deleted, as there are probably more and more devices coming? And the device types are listed under supported devices.

Reply:
I see what you mean. Yes, there are more devices coming, but to me this is actually a good reason to keep it like this. The idea here is to provide the user with information about the devices that the brand has, and after that there's the hass-specific supported devices, does that make sense to you=